internal/api/api.go

```diff
@@ -154,6 +154,7 @@ func NewAPIWithVersion(ctx context.Context, globalConfig *conf.GlobalConfigurati
 			r.With(sharedLimiter).Put("/", api.UserUpdate)
 
 			r.Delete("/identities/{identity_id}", api.DeleteIdentity)
+			r.Get("/identities/authorize", api.LinkIdentity)
```

Comment:
Swap these lines over? 
```suggestion
			r.Get("/identities/authorize", api.LinkIdentity)
			r.Delete("/identities/{identity_id}", api.DeleteIdentity)
```